ansible_runner/runner.py

```diff
@@ -80,7 +81,10 @@ def event_callback(self, event_data):
 
     def status_callback(self, status):
         self.status = status
-        status_data = dict(status=status, runner_ident=str(self.config.ident))
+        status_data = {
+            'status': status, 'runner_ident': str(self.config.ident),
+            'command': self.config.command, 'env': self.config.env, 'cwd': self.config.cwd
```

Comment:
Getting command/env/cwd are important, and this is a perfectly fine mechanism to get them. However, it doesn't seem right to send those on every status callback.

We need them on the "starting" status, I would prefer to not have them on running/successful/canceled.

Reply:
You are right, I forgot to make this conditional.